packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -601,12 +613,18 @@ export const iconMetadata: {
     IconTriangleTwoLinesHorizontal: {
         tags: ['collapse all']
     },
+    IconTwoChatSparkle: {
```

Comment:
Font Awesome [would call this](https://fontawesome.com/search?q=message&o=r&s=solid&ip=classic) `IconMessagesSparkle`, but we've used `Comment` for the squared off chat bubble in the past. 

@jattasNI - Should we duplicate the current nimble comment icon to fix the naming (it's actually a message icon), or something else?

Reply:
Looks like our use of the name "comment" [predates](https://github.com/ni/nimble/pull/368/files#diff-53037f6855f0b9895c5a3acc59ea6e11710c591927cd51db10f0bd0244d74e1e) our [requirement to follow FontAwesome's naming](https://github.com/ni/nimble/pull/1541). So since there's no longer a compelling argument for "comment" then yeah I'd vote to switch to "message" (with "comment" as a tag). I like the idea of keeping the legacy "comment" icon around for backwards compatibility until clients have a chance to update.

Reply:
I pushed a change to duplicate `comment` and `bell-and-comment` using the "message" terminology and marked the original ones with a "DEPRECATED" tag.

Reply:
Created https://github.com/ni/nimble/issues/2612 to track the tech debt in nimble. @sandeep-ni consider creating an AzDo tech debt issue to address this nimble issue.